### DISCUSSION TITLE
Pinter discussions (auto-generated message)

### DISCUSSION THREAD

**Discussion by adamhutchings** [plancklang/pinter#13] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by adamhutchings** [comment]
I write C code for pinter and am a primary maintainer of Planck.

**Comment by JakeRoggenbuck** [comment]
I'm also a primary maintainer and I help with the standard libs and the planning for that and the syntax of Planck.
